config/crd/bases/toolkit.image.fluxcd.io_imagepolicies.yaml

```diff
@@ -0,0 +1,94 @@
+
```

Comment:
Where did this file come from?

Reply:
Tutorial:

1) make a typo
2) generate the files
3) notice your typo and fix it
4) regenerate the files while not removing the old ones